R/test-elements.R

```diff
@@ -1030,11 +1038,11 @@ volume_calibration_page <- function(url, type = tools::file_ext(url),
 #'
 #' @param trigger_button_text (Character scalar) Text for the trigger button.
 #'
-#' @param failed_validation_message (Character scalar) Text to be displayed
-#' when validation fails.
+#' @param failed_validation_message (Named character vector) Text to be displayed
```

Comment:
The changes here break back-compatibility for people who used this argument previously when there was just one message to provide. I'd add some code to support such situations.

Reply:
Not sure how this breaks back-compatibility? Users of old code will have force_answer used as a logical, but then force_answer is set to "one", the old form (line 1087-1089). Or do I miss something?